### ISSUE TITLE
Hyperparameter Tuning & Optimization

### ISSUE THREAD

**Issue by rdargula** [Nisch9/Speech-Emotion-Recognition#23]
Tune batch size, learning rate, number of epochs, and dropout layers for better performance.
